### ISSUE TITLE
Spectral smoothing in cubeviz (cubeviz -> specviz)

### ISSUE THREAD

**Issue by brechmos-stsci** [spacetelescope/cubeviz#175]
I want to clarify several things here to do with the smoothing.  Currently @robelgeda has implemented both spatial and spectral smoothing as requested.  But specviz also does spectral smoothing (and is a large part of the functionality).  Some how it was missed that we had two versions of spectral smoothing.

So, after discussion with @kassin @bethanjames  and @robelgeda  it was decided:

1) Spatial smoothing

Spatial smoothing will remain where it currently is located and the functionality will all remain the same (as @robelgeda implemented it).

2) Spectral smoothing

Spectral smoothing will be *removed* from the "Data Processing" "Smoothing" dialog (which @robelgeda had created).  

Spectral smoothing will be done in specviz (both the preview and the application throughout the whole cube).  Specviz will propagate the spectrally smoothed dataset (ie after all spaxels have been spectrally smoothed).  

So, to summarize, spatial smoothing will stay where it is.  Spectral smoothing preview and application to all spaxels will happen in specviz.   

@kassin  @robelgeda Do you agree this is the summary and does it make sense?

Cc @nmearl 

Once agreed on, then @robelgeda will remove the spectral option from his panel. The spaxel application should be completed in #78

**Comment by kassin** [comment]
I agree

**Comment by nmearl** [comment]
To be honest, I thought this was the intent from the discussion last week -- but @robelgeda and I spoke this morning, and while it's certainly possible to keep all smoothing operations in CubeViz, it does seem strange to have to leave the SpecViz panel to do spectral operations (although, as discussed with Robel, it can be done!)

I'm glad we cleared this up! Thanks, @brechmos-stsci.

**Comment by robelgeda** [comment]
Yes, this is good.

**Comment by brechmos-stsci** [comment]
@robelgeda If you could use this ticket for the "removal" of spectral processing in your panel. Then create a PR and I'll check and merge.

**Comment by brechmos-stsci** [comment]
This change is now complete.
